pfrl/experiments/train_hrl_agent.py

```diff
@@ -0,0 +1,222 @@
+import logging
+import os
+from pfrl.agents.hrl.hiro_agent import HIROAgent
+
+from pfrl.experiments.evaluator import Evaluator
+from pfrl.experiments.evaluator import save_agent
+
+
+import os 
+import numpy as np
+
+
+def train_hrl_agent(
+    agent: HIROAgent,
+    env,
+    steps,
+    outdir,
+    checkpoint_freq=None,
+    max_episode_len=None,
+    step_offset=0,
+    evaluator=None,
+    successful_score=None,
+    step_hooks=(),
+    logger=None,
+    explore=True,
+    start_training_steps=100,
+    subgoal=None
+):
+
+    logger = logger or logging.getLogger(__name__)
+
+    episode_r = 0
+    episode_idx = 0
+
+    obs_dict = env.reset()
+    subgoal = subgoal
+
+    fg = obs_dict['desired_goal']
+    obs = obs_dict['observation']
```

Comment:
`obs_dict` should be `obs` instead of separating it in this scope. Instead, we can change behavior under the hood in agent functions so that we can utilize `train_agent.py`

Reply:
I think we should use `train_hrl_agent.py`. The behavior of the HRL agent is uniquely different than `train_agent.py`, and I think it would require an unnecessary amount of work to change the HRL agent to simply fit `train_agent.py`. Plus, there are other `train_agent_<something>.py` files that eliminate potential confusion.